### ISSUE TITLE
twitch.tv get stream url error

### ISSUE THREAD

**Issue by sysilion** [streamlink/streamlink-portable#12]
![image](https://cloud.githubusercontent.com/assets/2581852/23474460/362f3946-fef7-11e6-8cb6-292265abb1b7.png)

I got this error when I try to get the stream address of twitch.tv.

I've select flavor Portable EXE

**Comment by RosadinTV** [comment]
Hi, it seems like this issue is related to Pycountry.
Could you please download this file and extract to 'Releases\Streamlink\pycountry' folder ?
https://github.com/RosadinTV/Pycountry_TEST/raw/master/Pycountry%20Locales.zip

**Comment by sysilion** [comment]
![image](https://cloud.githubusercontent.com/assets/2581852/23488164/482b88e8-ff2e-11e6-875e-a8d4cb1abca6.png)

The same error still occurs.

**Comment by RosadinTV** [comment]
![twitch_streamlink](https://cloud.githubusercontent.com/assets/13382316/23489463/4af61abe-fed1-11e6-8542-68acffa85a0c.JPG)
Really strange, not sure if this a bug of pycountry or a bug with the twitch plugin (i think is something with pycountry because is working in my case but you we have different lenguage inputs)
@beardypig Do you have an idea of what can cause this issue?

**Comment by beardypig** [comment]
It appears to be a `streamlink` bug :( Sorry @sysilion it doesn't like the `ko_KR` locale. As a work-around you could set the locale to `en_US`, shouldn't affect too much.

```shell
"Python 3.5.2\python.exe" Streamlink\Streamlink.py --locale en_US twitch.tv/aaaaaa
```
And see if that works. I will work on a proper fix for that now.

**Comment by beardypig** [comment]
I have create a PR (streamlink/streamlink#650) on streamlink to fix this, but it probably won't be merged for a few hours.

**Comment by sysilion** [comment]
@beardypig i've test with your PR code :)
it works fine XD
@RosadinTV @beardypig thx for resolve problem.

![image](https://cloud.githubusercontent.com/assets/2581852/23507593/218c4584-ff91-11e6-8460-0b81001f66ad.png)

**Comment by beardypig** [comment]
Awesome :) No problem :)

**Comment by RosadinTV** [comment]
@beardypig Thanks, an amazing job.
PS: Do you know if the "locales" folder inside Pycountry have any real use in the Project?
Because it seems to work well without her (and 23 mb of size can be saved)
The Pycountry included in this portable repo didn't contains it but i want to be sure if a change is needed.

**Comment by beardypig** [comment]
I'm not sure :)

If you want to make it smaller you can use the two separate packages `iso3116` and `iso-639` which are much smaller, that's what the windows installer uses :)

**Comment by gravyboat** [comment]
I've merged #650 so I'm going to close this, thanks everyone for your input and work.
